<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCCCommunicationForm</class>
 <widget class="QDialog" name="RCCCommunicationForm">
  <property name="windowTitle">
   <string>Связь с RCC</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelRccName">
       <property name="text"><string>Наименование RCC:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="rccName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRccChannel">
       <property name="text"><string>Канал связи:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="rccChannel">
       <item><property name="text"><string>Email</string></property></item>
       <item><property name="text"><string>FTP</string></property></item>
       <item><property name="text"><string>VPN</string></property></item>
       <item><property name="text"><string>Web API</string></property></item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelMessageToRcc">
       <property name="text"><string>Сообщение:</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="messageToRcc"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="buttonSend">
       <property name="text"><string>Отправить</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text"><string>Закрыть</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>